plugin/src/main/scala/org/scalafmt/sbt/PluginInstanceSession.scala

```diff
@@ -0,0 +1,11 @@
+package org.scalafmt.sbt
+
+import java.nio.file.Path
+import org.scalafmt.interfaces.{Scalafmt, ScalafmtSession}
+
+class PluginInstanceSession(config: Path, instance: Scalafmt)
```

Comment:
1. should it be package-private?
2. perhaps rename to something like `LegacyScalafmtSession` or `CompatibilityScalafmtSession` or `EmulatedScalafmtSession`?